### ISSUE TITLE
Sitemap wird nicht berücksichtigt

### ISSUE THREAD

**Issue by codesache** [fritzmg/contao-simple-news-urls#21]
@fritzmg bei einer Contao 5.3.15 Installation wird die sitemap.xml im Root noch mit dem Ordner-Pfad der NewsDetailsSeite zusammen gesetzt. Gibt es in deiner Erweiterung dazu eine Einstellung die ich übersehen habe ?

Die URLs der NewsDetailsSeite werden nach der Umleitung deiner Erweiterung richtig im Root angezeigt.

**Comment by fritzmg** [comment]
Nein, die Sitemap Einträge werden aktuell nicht auf die Kurz-Variante geändert. Ist grundsätzlich auch nicht erforderlich, da es sowieso einen 301 redirect gibt (optimaler wäre es aber natürlich, wenn Crawler sich einen Request sparen könnten).

**Comment by fritzmg** [comment]
Implemented in `5.1.0`.
